examples/pylab_examples/annotation_demo.py

```diff
@@ -50,37 +50,39 @@
     s = np.cos(2*np.pi*t)
     line, = ax.plot(t, s, lw=3, color='purple')
 
-    ax.annotate('axes center', xy=(.5, .5), xycoords='axes fraction',
-                horizontalalignment='center', verticalalignment='center')
+    ax.annotate('figure pixels',
+                xy=(10, 10), xycoords='figure pixels')
 
-    ax.annotate('pixels', xy=(20, 20), xycoords='figure pixels')
+    ax.annotate('figure points', xy=(80, 80),
+                xycoords='figure points')
 
-    ax.annotate('points', xy=(100, 300), xycoords='figure points')
-
-    ax.annotate('offset', xy=(1, 1), xycoords='data',
-                xytext=(-15, 10), textcoords='offset points',
+    ax.annotate('point offset from data', xy=(2, 1),
+                xycoords='data',
+                xytext=(-15, 25), textcoords='offset points',
                 arrowprops=dict(facecolor='black', shrink=0.05),
                 horizontalalignment='right', verticalalignment='bottom',
                 )
 
-    ax.annotate('local max', xy=(3, 1), xycoords='data',
+    ax.annotate('axes fraction', xy=(3, 1), xycoords='data',
                 xytext=(0.8, 0.95), textcoords='axes fraction',
                 arrowprops=dict(facecolor='black', shrink=0.05),
                 horizontalalignment='right', verticalalignment='top',
                 )
 
-    ax.annotate('a fractional title', xy=(.025, .975),
+    ax.annotate('figure fraction', xy=(.025, .975),
                 xycoords='figure fraction',
                 horizontalalignment='left', verticalalignment='top',
                 fontsize=20)
 
     # use negative points or pixels to specify from right, top -10, 10
     # is 10 points to the left of the right side of the axes and 10
     # points above the bottom
-    ax.annotate('bottom right (points)', xy=(-10, 10),
-                xycoords='axes points',
-                horizontalalignment='right', verticalalignment='bottom',
-                fontsize=20)
+    ax.annotate('pixel offset from axes fraction', xy=(1, 0),
```

Comment:
(continued)  that example will put text that is right algined starting at 20px to the left of left edge of the axes.  The example in the PR puts the text 20px to the left of the right edge of the axes.